### PR TITLE
Bl adding ajax

### PR DESCRIPTION
**Pull request by tluu5**

### CODE REVIEW

File: app/views/comments/destroy.js.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+$("#<%= dom_id(@comment) %>").fadeOut(2000, function() {
```
Comment (aldoportillo) [inline app/views/comments/destroy.js.erb:1]: nice job using this.

**Review by aldoportillo — CHANGES_REQUESTED** [review]
We are missing like and follow functionality

File: app/views/comments/_form.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<li class="list-group-item">
-  <%= form_with(model: comment) do |form| %>
+<li id="<%= dom_id(comment.photo) %>_<%= dom_id(comment) %>_form" class="list-group-item">
```
Comment (jessjno) [inline app/views/comments/_form.html.erb:1]: good job using dom_id

File: app/views/follow_requests/_follow_unfollow.html.erb
Comment (jessjno) [inline app/views/follow_requests/_follow_unfollow.html.erb:1]: good job replacing turbo_method

File: app/views/follow_requests/create.js.erb
Comment (jessjno) [inline app/views/follow_requests/create.js.erb:1]: good job using camel case

**Review by jessjno — APPROVED** [review]

**Pull request merged**